templates/web/pages/product/includes/product_image.tt.html

```diff
@@ -7,8 +7,8 @@
 		<p>[% creativecommons %][% original_link %]</p>
 		<p>[% attribution %]</p>
 	</figcaption>
-	<a class="button small action-report_product_to_nutripatrol" href="https://nutripatrol.openfoodfacts.org/flag/image/?barcode=[% code %]&image_id=[% original_id %]&source=web&flavor=off">
-		Signaler ce produit à nos modérateurs
+	<a class="button small action-report_product_to_nutripatrol" href="https://nutripatrol.openfoodfacts.org/flag/image/?barcode=[% code %]&image_id=[% original_id %]&source=web&flavor=[% flavor %]">
```

Comment:
It would be great to add a test in tests/integration/web_html.t to test this page